integration/form-test.ts

```diff
@@ -921,7 +921,7 @@ test.describe("Forms", () => {
           await app.goto("/projects/blarg");
           let html = await app.getHtml();
           let el = getElement(html, `#${SPLAT_ROUTE_NO_ACTION}`);
-          expect(el.attr("action")).toMatch("/projects");
+          expect(el.attr("action")).toMatch("/projects/blarg");
```

Comment:
These were asserting the buggy behavior fixed by https://github.com/remix-run/react-router/pull/10983